### PR TITLE
chore(deps): bump risc0 to v0.19

### PR DESCRIPTION
**Pull request by Wollac**

### CODE REVIEW

**Review by intoverflow — APPROVED** [review]

File: testing/ef-tests/tests/executor.rs [hunk 1]
```diff
@@ -75,11 +72,11 @@ fn executor(
         let env = ExecutorEnv::builder()
             .session_limit(None)
             .segment_limit_po2(SEGMENT_LIMIT_PO2)
-            .add_input(&to_vec(&chain_spec).unwrap())
-            .add_input(&to_vec(&input).unwrap())
+            .write_slice(&to_vec(&chain_spec).unwrap())
+            .write_slice(&to_vec(&input).unwrap())
```
Comment (flaub) [inline testing/ef-tests/tests/executor.rs:76]: This can be:
```
.write(&chain_spec)
.write(&input)
```

File: host/tests/blocks.rs [hunk 2]
```diff
@@ -61,10 +61,10 @@ fn empty_blocks(#[files("testdata/ethereum/*.json.gz")] path: PathBuf) {
     let env = ExecutorEnv::builder()
         .session_limit(None)
         .segment_limit_po2(20)
-        .add_input(&to_vec(&input).unwrap())
+        .write_slice(&to_vec(&input).unwrap())
```
Comment (flaub) [inline host/tests/blocks.rs:64]: Suggestion:
```
.write(&input)
```

File: testing/ef-tests/tests/executor.rs (hunk 1 above)
Comment (Wollac) [reply testing/ef-tests/tests/executor.rs:76]: I considered this, but then you still need to unwrap and formatted it gets
```rust
.write(&chain_spec)
.unwrap()
.write(&input)
.unwrap()
```
which I felt was a bit harder to read. But I don't have a strong opinion on this 🤷

Comment (flaub) [reply testing/ef-tests/tests/executor.rs:76]: If there's no strong need to go either way, I'd default to having our examples and demos to be consistent (and for future host/guest I/O explainer docs).

**Pull request merged**